### PR TITLE
Make toolbar actions lift the 'text changed' flag

### PR DESCRIPTION
**Pull request by chekoopa**
Improves `text not saved` warning feature, adds two global variables for other modules (`hyphaChanged`, `hyphaOriginal`).

### CODE REVIEW

File: static/editor.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
 (function () {
-	let changed = false;
+	window.hyphaChanged = false;
 	let textarea = document.querySelector('.edit-form__textarea');
 	let form = document.querySelector('.edit-form');
 
+	// some browsers tend to cache user's text, so we'd better keep
+	// the original one for later use (e.g. draft notifications)
+	window.hyphaOriginal = textarea.value;
```
Comment (handlerug) [inline static/editor.js:8]: Why do we need this variable?

Comment (chekoopa) [reply static/editor.js:8]: A little feature for draft support. I can cut it and present in another PR if it's desired.

Comment (handlerug) [reply static/editor.js:8]: Yeah, would've been better to not add more code until it's needed and keep the PRs atomic.

Comment (chekoopa) [reply static/editor.js]: Force-pushed the fix

**Review by handlerug — APPROVED** [review]
LGTM

**Pull request merged**